connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientDataFrameStatSuite.scala

```diff
@@ -248,19 +248,19 @@ class ClientDataFrameStatSuite extends RemoteSparkSession {
 
   test("Bloom filter test invalid inputs") {
     val df = spark.range(1000).toDF("id")
-    val message1 = intercept[SparkException] {
+    val message1 = intercept[AnalysisException] {
       df.stat.bloomFilter("id", -1000, 100)
     }.getMessage
-    assert(message1.contains("Expected insertions must be positive"))
+    assert(message1.contains("VALUE_OUT_OF_RANGE"))
```

Comment:
Could you just invoke the `getErrorClass()` method, please.

Reply:
+1